v3/cypress/e2e/map.spec.ts

```diff
@@ -0,0 +1,198 @@
+    c.createFromToolshelf("map")
+    c.getComponentTitle("map").should("have.text", componentName)
+    c.changeComponentTitle("map", newComponentName)
+    c.getComponentTitle("map").should("have.text", newComponentName)
+    
+    cy.log("creates maps with new component name")
+    c.createFromToolshelf("map")
+
+    c.getComponentTitle("map").should("contain", componentName)
+    c.getComponentTitle("map", 1).should("contain", componentName)
+  })
+  it("checks all map tooltips", () => {
+    cfm.openLocalDoc(filename1)
+    c.createFromToolshelf("map")
+    c.selectTile("map", 0)
+
+    c.getToolShelfIcon("map").then($element => {
+      c.checkToolTip($element, c.tooltips.mapToolShelfIcon)
+    })
+    c.getMinimizeButton("map").then($element => {
+      c.checkToolTip($element, c.tooltips.minimizeComponent)
+    })
+    c.getCloseButton("map").then($element => {
+      c.checkToolTip($element, c.tooltips.closeComponent)
+    })
+    map.getZoomInButton().then($element => {
+      c.checkToolTip($element, c.tooltips.mapZoomInButton)
+    })
+    map.getZoomOutButton().then($element => {
+      c.checkToolTip($element, c.tooltips.mapZoomOutButton)
+    })
+    map.getResizeIcon().then($element => {
+      c.checkToolTip($element, c.tooltips.mapResizeButton)
+    })
+    map.getHideShowButton().then($element => {
+      c.checkToolTip($element, c.tooltips.mapHideShowButton)
+    })
+    map.getDisplayValuesButton().then($element => {
+      c.checkToolTip($element, c.tooltips.mapDisplayValuesButton)
+    })
+    map.getDisplayConfigButton().then($element => {
+      c.checkToolTip($element, c.tooltips.mapDisplayConfigButton)
+    })
+    map.getCameraButton().then($element => {
+      c.checkToolTip($element, c.tooltips.mapCameraButton)
+    })
+  })
+  it("checks numerical and categorical attributes for map legend", () => {
+    cfm.openLocalDoc(filename2)
+
+    mlh.verifyLegendLabel(arrayOfAttributes[0])
+    mlh.verifyCategoricalLegend(arrayOfValues[0].values.length)
+    mlh.selectCategoryNameForCategoricalLegend(arrayOfValues[0].values[0])
+    mlh.verifyCategoricalLegendKeySelected(arrayOfValues[0].values[0], 89)
+    mlh.selectCategoryNameForCategoricalLegend(arrayOfValues[0].values[1])
+    mlh.verifyCategoricalLegendKeySelected(arrayOfValues[0].values[1], 67)
+    mlh.selectCategoryColorForCategoricalLegend(arrayOfValues[0].values[0])
+    mlh.verifyCategoricalLegendKeySelected(arrayOfValues[0].values[0], 89)
+    mlh.selectCategoryColorForCategoricalLegend(arrayOfValues[0].values[1])
+    mlh.verifyCategoricalLegendKeySelected(arrayOfValues[0].values[1], 67)
+
+    // TODO: unselecting by clicking away in map doesn't work
+    // Uncomment the below two lines once it's fixed
+    // mlh.unselectLegendCategory()
+    // mlh.verifyNoLegendCategorySelectedForCategoricalLegend()
+
+    // TODO: This drag legend doesn't work for maps
+    // Uncomment line 95 and remove lines 96, 97 once it starts working
```

Comment:
This is picky, sorry, but would it be better to word these instructions more like you do elsewhere, something like "Uncomment the next line and remove the two after it" just in case the line numbers change?